src/components/core/typography.tsx

```diff
@@ -23,42 +25,45 @@ export const ExternalLink = ({
   )
 }
 
-const Heading = styled.h2`
-  display: flex;
-  align-items: center;
-  margin: 0;
-`
-
-export const PageTitle = ({
+type PageHeaderProps = {
+  title: string
+  icon?: React.Node
+  subTitle?: React.Node
+  children?: ReactNode
+}
+export const PageHeader = ({
+  title,
   icon,
-  children,
-  extra,
-  style
-}: {
-  children: any
-  icon?: any
-  extra?: any
-  style?: any
-}) => {
+  subTitle,
+  children
+}: PageHeaderProps) => {
+  const content = children || title
+  usePageTitle(title)
   return (
-    <Heading style={{ paddingBottom: '1rem', ...style }}>
+    <Heading>
       {icon && <PageTitleIcon>{icon}</PageTitleIcon>}
-      <div>
-        {children}
-        {extra && <PageTitleExtra>{extra}</PageTitleExtra>}
-      </div>
+      {content}
+      {subTitle && <PageSubTitle>{subTitle}</PageSubTitle>}
     </Heading>
   )
 }
 
+const Heading = styled.h1`
+  display: flex;
+  align-items: center;
+  margin: 0;
+  font-size: 1.5rem;
+  padding-bottom: 0.5rem;
+`
+
 const PageTitleIcon = styled.div`
   margin-right: 0.5rem;
   color: var(--iconColor);
   display: flex;
 `
 
-const PageTitleExtra = styled.span`
-  color: var(--textSecondary);
-  font-size: 16px;
-  margin-left: 0.25rem;
+const PageSubTitle = styled.span`
```

Comment:
I just noticed that this has low colour contrast (doesn't meet AA). We might wanna make it a bit darker?

![Screenshot from 2021-06-15 13-27-44](https://user-images.githubusercontent.com/7598058/121988430-9e2abe80-cddd-11eb-9db3-80a6abaa520c.png)

Reply:
Thank you Benjamin, i will fix that, maybe make it darker and smaller then, since I wanted the sub title to less visible than the title, like what Ant D does.

![image](https://user-images.githubusercontent.com/5546996/121996263-3e371680-cde3-11eb-866c-22851079befc.png)

Reply:
That sounds good

Reply:
Sorry for the late reply @BBlackwo

What do you think of this:

- same size
- darker color
- using a dot `•` as a separator

![image](https://user-images.githubusercontent.com/5546996/124343820-7d4dd000-dc09-11eb-8458-1bde4f8b5512.png)